src/java/org/apache/cassandra/metrics/UnweightedCacheMetrics.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.cassandra.metrics;
+
+import com.codahale.metrics.Gauge;
+import org.apache.cassandra.cache.UnweightedCacheSize;
+
+import static org.apache.cassandra.metrics.CassandraMetricsRegistry.Metrics;
+
+/**
+ * Metrics for an unweighted cache like {@code AuthCache}.
+ */
+public class UnweightedCacheMetrics extends AbstractCacheMetrics
+{
+    /**
+     * Cache capacity (maximum number of entries)
+     */
+    public final Gauge<Integer> maxEntries;
+
+    /**
+     * Total number of cache entries
+     */
+    public final Gauge<Integer> entries;
+
+    /**
+     * Create metrics for given cache.
+     *
+     * @param type  Type of Cache to identify metrics
+     * @param cache Unweighted Cache to measure metrics
+     */
+    public UnweightedCacheMetrics(String type, UnweightedCacheSize cache)
+    {
+        super(new DefaultNameFactory("UnweightedCache", type), type);
+
+        maxEntries = Metrics.register(factory.createMetricName("Capacity"), cache::maxEntries);
```

Comment:
Is this change only internally visible? If it is exposed to clients I think we need to keep it like it is to not break anything.

Reply:
This will be exposed to an end user, however, as we don't have such metrics in the public API at this moment, we can expose it with the "right" name. Even though auth caches and weighted caches have different MBeans, vtables there is no need to keep the attribute names in sync between them, so I guess we can change it.

Reply:
@Mmuzaf I would rather stay consistent and renam `maxEntries` gauge to `capacity` to be on par with `CacheMetrics` which were there first which have it `capacity` too. There is no reason to have this naming discrepancy between different types of cache metrics and term "capacity" is just perfectly fine.

Reply:
you know what ... you are right. `Capacity` here should be `MaxEntries`. Because `Capacity` and `MaxEntries` are not the same thing.